getting_started/setup_vm/roles/openenclave/tasks/binary_install.yml

```diff
@@ -4,4 +4,82 @@
 - name: Install Open Enclave
   apt:
     deb: "{{ oe_deb }}"
-  become: true
\ No newline at end of file
+  become: yes
+
+- name: Make OE LVI dir
+  file:
+    path: "{{ oe_lvi_bin_dir }}"
+    state: directory
+  become: yes
+
+- name: Copy invoke_compiler to OE LVI dir
+  copy:
+    remote_src: yes
+    src: "{{ oe_lvi_scripts_dir }}/invoke_compiler"
+    dest: "{{ oe_lvi_bin_dir }}/invoke_compiler"
+    mode: "0555"
+  become: yes
+
+- name: Create OE LVI symlink to clang-8
```

Comment:
I suggest lifting 8 to a clang_version under vars/, to make future upgrades easier.